Formula/llvm.rb

```diff
@@ -381,8 +402,21 @@ def caveats
               "-I#{MacOS.sdk_path}/usr/include",
               "-L#{lib}",
               "-Wl,-rpath,#{lib}", "test.cpp", "-o", "test"
-      assert_includes MachO::Tools.dylibs("test"), "#{opt_lib}/libc++.1.dylib"
+      assert_includes MachO::Tools.dylibs("test"), "@rpath/libc++.1.dylib"
       assert_equal "Hello World!", shell_output("./test").chomp
+
+      # link against installed libc++experimental
+      if build.with? "experimental"
+        system "#{bin}/clang++", "-v", "-nostdinc",
+                "-std=c++11", "-stdlib=libc++",
+                "-lc++experimental",
+                "-I#{opt_include}/c++/v1",
+                "-I#{libclangxc}/include",
+                "-I#{MacOS.sdk_path}/usr/include",
+                "-L#{lib}",
+                "-Wl,-rpath,#{lib}", "test_experimental.cpp", "-o", "test_experimental"
+        assert_equal "\"#{Pathname.pwd}\"", shell_output("./test_experimental").chomp
```

Comment:
Well, in `test_experimental` I actually call cwd, so in test I compare current working directories, if `testpath` happens to be the same as current working directory is good, but my guess it might change in future? For me current implementation is more bullet proof, but I am ok to change it to suggested if you insist.